### ISSUE TITLE
Sheet keeps trying to initialize after it's been set up

### ISSUE THREAD

**Issue by N3w2c0d3** [boilermaker-ben/Google-Sheets-NFL-Pick-Ems-and-Survivor-Pool#14]
I've followed the promits and had everything working for week one and the start of week two.  However, when I logged in today to import the picks, the "picks" tab only shows the initialize option (see screenshot) and if I click it, the file tries to set the file sheet up like its the first time.  I have not run that script as I do not want to override the week 1 information.

<img width="755" alt="Screenshot 2024-09-15 at 16 19 47" src="https://github.com/user-attachments/assets/dc21686a-9710-4d2f-aeeb-1cc1c8d6c05f">

<img width="934" alt="Screenshot 2024-09-15 at 16 20 42" src="https://github.com/user-attachments/assets/1b1317ed-b5a3-4afb-97c4-25e019f6b073">

Any ideas?

**Comment by boilermaker-ben** [comment]
You're only the third person who's had an issue with this, so it isn't widespread, but clearly there's some kind of double check that I need to put in place. Thanks for posting here and also for using the sheet/script!

To correct this, when using a desktop browser, please go to your **Extensions** > **Apps Script** window. Then go to the gear icon on the left menu bar to access the **Project Settings**. Scroll down to the bottom of that area and there are **Script Properties** that should be set.  You should "Edit Script Properties" at the bottom and add a property called **initialized** that has a value of **true**. That should fix this for any future refreshes (it's a variable that should have been set when you initialized the sheet the first time).

**Comment by NadlesKVs** [comment]
Wanted to chime in and mention that I did have this issue as well. Not sure what I did to fix it but it did stop after week 1.

Also it kept locking members on me when I launched it but after Week 1 it hasn't been an issue for me.

**Comment by boilermaker-ben** [comment]
Thanks for chiming in.

When you say that it was "locking members", did it actually have a pop-up
saying membership was locked or did it just have the menu up that would
prompt you to "unlock" members? (Maybe it was just loading the wrong menu,
rather than actually locking membership, as that would also entail a
removal of the "New User" from the form)

**Comment by NadlesKVs** [comment]
It did not say membership was locked. The menu was loading incorrectly, but when I went to import picks that included new members, it wouldn't actually add them as new members. It would just cycle asking if I wanted to add the new members IIRC.

I would have to unlock membership when I opened the sheet, import new picks which would add the new users correctly, and everything was fine after that.

Either way everything is working great now for us.
